pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -142,6 +142,8 @@ type CanaryStrategy struct {
 	// at any time during the update is atmost 130% of original pods.
 	// +optional
 	MaxSurge *intstr.IntOrString `json:"maxSurge,omitempty"`
+	// ParellelAnalysis runs a separate analysisRun while all the steps execute. This is intended to be a continuous validation of the new ReplicaSet
+	ParellelAnalysis *RolloutAnalysisRun `json:"parellelAnalysis,omitempty"`
```

Comment:
parellel is misspelled, but i think the field should be renamed to just `analysis` anyways.